src/5e-SRD-Levels.json

```diff
@@ -6887,9 +6887,9 @@
     "prof_bonus": 3,
     "feature_choices": [
       {
-        "index": "additional-eldritch-invocation-1",
-        "name": "Additional Eldritch Invocation",
-        "url": "/api/features/additional-eldritch-invocation-1"
+        "index": "eldritch-invocations-2",
```

Comment:
Should these be off by one compared to the original indices? I'm sorry I'm on my phone or I would be able to check properly.

Reply:
> Should these be off by one compared to the original indices? I'm sorry I'm on my phone or I would be able to check properly.

They should be. The first eldritch invocations feature was just named `eldritch-invocations`. It is now `eldritch-invocations-1` and all subsequent additional eldritch invocations follow the same naming scheme, hence the incriment.